.ci/ci_cache_kernel.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+function handle_error {
```

Comment:
There seems to be quite a bit of duplication between this PR and #1215. Could you maybe refactor the common bits into `.ci/lib.sh`?

Reply:
agree, lets merge #1215 first, I will label this with `DNM`, once is merged I'll refactor, I dont want to move both changes in the same PR.

Reply:
Hi @jcvenegas - https://github.com/kata-containers/tests/pull/1215 has now landed.